custom/icds_reports/ucr/tests/test_expressions.py

```diff
@@ -174,3 +174,22 @@ def test_no_to_statement(self):
 
         self.assertEqual(len(forms), 0)
         self.assertEqual(forms, [])
+
+
+class TestGetAppVersion(SimpleTestCase):
+    def test_cases(self):
+        expression = ExpressionFactory.from_spec({
+            "type": "icds_get_app_version",
+            "app_version_string": {
+                "type": "property_name",
+                "property_name": "app_version_string",
+            },
+        })
+        self.assertEqual(None, expression({"app_version_string": "bar"}))
+        self.assertEqual(None, expression({}))
+        self.assertEqual(None, expression({"app_version_string": ""}))
+        self.assertEqual(9969, expression({
+            "app_version_string": "CommCare Android, version 2.36.2(433756). App v9969. "
+                                  "CommCare Version 2.36. Build 433756, built on: 2017-06-23"}))
+        self.assertEqual(105, expression({
+            "app_version_string": "CommCare Android, version 2.36.2(433756). App v105."}))
```

Comment:
Why is this string half the lenght of the previous one? (without the commcare version info)